Resources/Prototypes/_Wega/Roles/Jobs/Specific/prisoner.yml

```
@@ -0,0 +1,20 @@
+- type: job
+  id: Prisoner
+  name: job-name-prisoner
+  description: job-description-prisoner
+  playTimeTracker: JobPrisoner
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Security
+      time: 72000 #20h
+  startingGear: PrisonerGear
+  icon: "JobIconPrisoner"
+  canBeAntag: true
```

Какой антаг в камере, тупо же
```suggestion
  canBeAntag: false
```